sismic/interpreter/default.py

```diff
@@ -610,7 +610,12 @@ def _create_stabilization_step(self, names: Iterable[str]) -> Optional[MicroStep
                 return MicroStep(entered_states=states_to_enter, exited_states=[leaf.name])
             elif isinstance(leaf, OrthogonalState) and self._statechart.children_for(leaf.name):
                 return MicroStep(entered_states=sorted(self._statechart.children_for(leaf.name)))
-            elif isinstance(leaf, CompoundState) and leaf.initial:
+            elif isinstance(leaf, CompoundState):
+                if leaf.initial is None:
+                    return MicroStep(entered_states=[self._statechart.children_for(leaf.name)[0]])
```

Comment:
I really want to avoid using an implicit order (such as document order) for execution semantics. This makes sismic more "predictable", even if we move away from the scxml philosophy for the moment.

Reply:
Understood I do agree explicitly declaring and initial state or initial transition leaves no ambiguity as to which state is entered when the composite state is the target of a transition.